### PR TITLE
add ecs command for aws

### PR DESCRIPTION
**Pull request by pathcl**
Hello!

As an ECS user I'd like to have covered common operations: restart, list services, scale, rollback, etc. For now I'm including the first two.

### CODE REVIEW

File: command/clouds/aws/ecs.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+	var text string
+
+	text += "Hello <@" + message.User + ">, current services:. \n"
+
+	for _, v := range svc {
+		text += fmt.Sprintf("• %s", v)
+		text += "\n"
+	}
+
+	// t.SendEphemeralMessage(message, text)
```
Comment (brainexe) [inline command/clouds/aws/ecs.go:57]: you can remove this line :)

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,5 @@
 /build/
 /vendor/
 /config.yaml
-/storage/
\ No newline at end of file
+/storage/
+config.yaml
```
Comment (brainexe) [inline .gitignore:7]: `/config.yaml` is on the list. are both needed?

File: command/clouds/aws/ecs.go [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+	if _ecs == nil {
+		_ecs = ecs.New(session.New(getServiceConfiguration()))
+	}
+	return _ecs
+}
+
+func getServiceConfiguration() *aws.Config {
+	region := os.Getenv("AWS_REGION")
+	if region == "" {
+		region = "us-east-1"
```
Comment (brainexe) [inline command/clouds/aws/ecs.go:127]: maybe also add a log entry if no AWS_REGION is set. else I'd expect some debug hell

Comment (pathcl) [reply command/clouds/aws/ecs.go:124]: added log.Info couldn't find a way to make it debug

**Comment by pathcl** [comment]
We're missing docs for this feature along with tests. Is localstack an option?

**Review by brainexe — APPROVED** [review]

**Pull request merged**